docs/install/install-server-with-docker/install-server.inc

```diff
@@ -48,6 +50,11 @@ Here is a minimalist docker-compose file content that can be used to deploy the
                 config:
                     - subnet: 172.30.0.0/16
 
+This docker-compose file uses the following config file to overwrite the default config that came with the rpm:
+
+.. include:: install-server-with-docker/default-server-config.cfg
+    :literal:
+
```

Comment:
If we add this, we will have the same config file in two different locations. Once in the `inmanta` repository and once in the `inmanta-core` repository. Over time, we will update one config file and forget to update the other one. This will cause confusion as users will rely on the fact that the documentation is accurate. Maybe it's sufficient to mention that the default config, included in the docker image, is as such that it connects to a database with hostname `iso_postgres`, to make sure it works correctly with the above-mentioned docker compose file and that it should be changed if a different setup is desired. We also don't mention the default config of an RPM install on the documentation pages. We just mention where the user can find the file.

Reply:
I added a short explanation in https://github.com/inmanta/inmanta-core/pull/3852/commits/e8eda061a0dd103e92da5e93a448503d9a2b515f